packages/components/src/lib/AppMapRPC.ts

```diff
@@ -117,11 +118,15 @@ export default class AppMapRPC {
 
   appmapStats(): Promise<any> {
     return new Promise((resolve, reject) => {
-      this.client.request('appmap.stats', {}, function (err: any, error: any, result: any) {
-        if (err || error) return reportError(reject, err, error);
+      this.client.request(
+        AppMapRpc.Stats.V2.Method,
```

Comment:
Should we consider this a breaking change due to reliance on the v2 API? The index process doesn't bind the v2 stats handler (though maybe it should).

Reply:
But the index process also expects "one process per appmap.yml" so v2 stats isn't really meaningful in that scenario.